integreat_cms/static/src/js/media-management/component/breadcrumbs.tsx

```diff
@@ -37,7 +72,28 @@ export default function Breadcrumbs({ breadCrumbs, searchQuery, mediaTranslation
                 <li class="py-2">
                   <ChevronRight />
                 </li>
-                <li>
+                <li
+                  className={currentDragTarget === directory.id ? "text-blue-600 text-2xl" : ""}
+                  onDragOver={(e) => {
+                    if (allowDrop) {
+                      e.preventDefault();
+                      setCurrentDragTarget(directory.id);
+                    }
+                  }}
+                  onDragLeave={(e) => {
+                    if (allowDrop) {
+                      e.preventDefault();
+                      setCurrentDragTarget(null);
+                    }
+                  }}
+                  onDrop={(e) => {
+                    if (allowDrop) {
+                      e.preventDefault();
+                      setCurrentDragTarget(null);
+                      dropItem(directory.id);
+                    }
+                  }}
+                >
                   <Link
                     href={`/${directory.id}/`}
                     class="block hover:bg-water-600 px-3 py-2 rounded break-all"
```

Comment:
```suggestion
                    class="flex flex-wrap hover:bg-water-600 px-3 py-2 rounded break-all"
```
And maybe you could add
```
                    {allowDrop && (currentDragTarget === directory.id ? (
                      <FolderOpen className={"mr-2"} />
                    ) : (
                      <Folder className={"mr-2"} />
                    ))}
```
before the `{directory.name}``(I cannot make an automated suggestion here)